lib/spack/docs/basic_usage.rst

```diff
@@ -2,7 +2,7 @@
 
 Basic usage
 =====================
-
+p
```

Comment:
You can do that, or what I usually do is clone your fork of Spack, check out the PR, and change it with `commit --amend`